src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxInvoicePluginApi.java

```diff
@@ -67,7 +66,7 @@ public AvaTaxInvoicePluginApi(final AvaTaxConfigurationHandler avaTaxConfigurati
 
     @Override
     public List<InvoiceItem> getAdditionalInvoiceItems(final Invoice invoice, final boolean dryRun, final Iterable<PluginProperty> properties, final CallContext context) {
-        final Collection<PluginProperty> pluginProperties = new ArrayList<PluginProperty>(ImmutableList.<PluginProperty>copyOf(properties));
+        final Collection<PluginProperty> pluginProperties = Lists.newArrayList(properties);
```

Comment:
Can you keep the explicit type argument (i.e. `Lists.<PluginProperty>newArrayList(properties)`)? Type inference is broken on old 1.6 jdk versions.